README.md

```diff
@@ -1 +1,29 @@
 # StardewValleyPocketTracker
+## Purpose
+This is an android application that is being created to gain experience developing applications
+in android studio, using git, solidifying programming fundamentals, and designing an application
+from markup to deployment.
+
+## Description
+This is a fan-made project.
+Stardew Valley Pocket tracker keeps track of calender events in Stardew Valley.
+The app also keeps track of items that the user has acquired.
+
+
+## Visuals
+<insert some images of blueprint? or mockup?>
+
+## Installation
+Android version 4.0.3 (Ice Cream Sandwich) or higher is required.
```

Comment:
If this is not intended for the play store, add information about pulling the code, building it locally, enabling developer options on your device, and installing the APK file.

If it is intended for the play store, put a comment about downloading it via the playstore (with a placeholder link) here.

Reply:
intended for app store, will include placeholder link